doc/tutorials/cloud/01_import_data.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<p>This needs to go to the notebook connector as a function and be connected to the secret store. We will need this for users anyway, if they access their own data, also if our data is publicly available.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Most likely this will go away completely after my cloud-storage-extension public bucket fix</p>